### PR TITLE
Doc: update README.md for CloudberryDB

### PR DESCRIPTION
**Pull request by tuhaihe**
---

### Change logs

Add the several ecosystem project information for Cloudberry Database, including the website, gpbackup, Bootcamp program, gp-common-go-libs, PostGIS, MADlib, PXF, PL/R, and PL/Java.

Also, update the logo files to support both light and dark system themes.

### Why are the changes needed?

_Describe why the changes are necessary._

### Does this PR introduce any user-facing change?

_If yes, please clarify the previous behavior and the change this PR proposes._

### How was this patch tested?

_Please detail how the changes were tested, including manual tests and any relevant unit or integration tests._

### Contributor's Checklist

Here are some reminders and checklists before/when submitting your pull request, please check them:

- [x] Make sure your Pull Request has a clear title and commit message. You can take [git-commit](https://github.com/cloudberrydb/cloudberrydb/blob/main/.gitmessage) template as a reference.
- [x] Sign the Contributor License Agreement as prompted for your first-time contribution(*One-time setup*).
- [x] Learn the [coding contribution guide](https://cloudberrydb.org/contribute/code), including our code conventions, workflow and more.
- [ ] List your communication in the [GitHub Issues](https://github.com/cloudberrydb/cloudberrydb/issues) or [Discussions](https://github.com/orgs/cloudberrydb/discussions) (if has or needed).
- [ ] Document changes.
- [ ] Add tests for the change
- [ ] Pass `make installcheck`
- [ ] Pass `make -C src/test installcheck-cbdb-parallel`
- [ ] Feel free to request `cloudberrydb/dev` team for review and approval when your PR is ready🥳

### CODE REVIEW

**Comment by tuhaihe** [comment]
Do not merge this pull request unless the other ecosystem project repositories are made publicly accessible.

**Comment by tuhaihe** [comment]
> Do not merge this pull request unless the other ecosystem project repositories are made publicly accessible.

The other projects require more work to compile and build, so we will simply add the existing public repos for now. This PR can start being reviewed now.

**Review by avamingli — APPROVED** [review]

**Pull request merged**
